pkg/placement/plugins/steady/steady.go

```diff
@@ -46,6 +49,13 @@ func (s *Steady) Description() string {
 
 func (s *Steady) Score(
 	ctx context.Context, placement *clusterapiv1beta1.Placement, clusters []*clusterapiv1.ManagedCluster) (plugins.PluginScoreResult, *framework.Status) {
+	startTime := time.Now()
```

Comment:
if this is the common thing, could we move this to the scheduler?

Reply:
yes, moved to scheduler.